packages/hardhat-chai-matchers/src/internal/changeEtherBalance.ts

```diff
@@ -14,20 +15,28 @@ export function supportChangeEtherBalance(Assertion: Chai.AssertionStatic) {
       options?: BalanceChangeOptions
     ) {
       const { BigNumber } = require("ethers");
+
+      // see buildAssert's jsdoc
```

Comment:
Done in 34fe9d5b808d6e77693ac21d0bcd85c5f992621d, but I used a semicolon instead of a period just because I once read [this comic](https://theoatmeal.com/comics/semicolon).